apps/backend/src/schema/schema.ts

```diff
@@ -0,0 +1,52 @@
+import { z } from 'zod';
+
+const addressSchema = z.object({
+    address: z.string().min(1),
+    city: z.string().min(1),
+    state: z.string().min(1),
+    zipcode: z.number(),
+})
+
+// Regex to check for phone numbers formatted with `-`, ' ', and '.' separators, 
+// and with and without area code surrounded by parenthesis
+const phoneNumberRegex = RegExp('`^(\+\d{1,2}\s)?\(?\d{3}\)?[\s.-]\d{3}[\s.-]\d{4}$`')
+const phoneNumber = z.string().regex(phoneNumberRegex)
+
+// Part of form to be filled out by the child's parent/legal guardian
+const guardianFormSchema = z.object({
+    childsName: z.string().min(1),
+    ssn: z.number(),
```

Comment:
oops sorry missed this the first time, ssn is probably also a string? idk if ssn's could start with 0's

Reply:
Good catch